indexing-service/src/test/java/io/druid/indexing/overlord/autoscaling/EC2AutoScalerSerdeTest.java

```diff
@@ -28,13 +29,56 @@
 import com.google.common.io.BaseEncoding;
 import io.druid.indexing.overlord.autoscaling.ec2.EC2AutoScaler;
 import io.druid.jackson.DefaultObjectMapper;
+import org.easymock.EasyMock;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class EC2AutoScalerSerdeTest
 {
+  private AmazonEC2Client amazonEC2Client = EasyMock.createMock(AmazonEC2Client.class);
+  private InjectableValues mockInjectableValues = new InjectableValues()
+  {
+    @Override
+    public Object findInjectableValue(
+        Object valueId, DeserializationContext ctxt, BeanProperty forProperty, Object beanInstance
+    )
+    {
+      if (valueId == "com.amazonaws.services.ec2.AmazonEC2") {
+        return amazonEC2Client;
+      } else {
+        return null;
```

Comment:
Is there anything else that gets injected? can this just throw an error if something else is requested?